stackslib/src/net/server.rs

```diff
@@ -517,17 +528,17 @@ impl HttpPeer {
 
                 debug!("HTTP event {} connected ({:?})", event_id, &data_url);
 
-                if let Err(_e) = self.register_http(
+                if let Err(e) = self.register_http(
                     network_state,
                     node_state,
                     *event_id,
                     socket,
                     data_url.clone(),
                     initial_request_opt,
                 ) {
-                    debug!(
-                        "Failed to register HTTP connection ({}, {:?})",
-                        event_id, data_url
+                    info!(
```

Comment:
How often will this log line show up in a node?

Reply:
It hasn't shown up at all in my production node (sampled over an 18.6 hour period).